### PR TITLE
added akshay sharma profile to the members

### LINKED ISSUE #5033: [Community] Member Profile: Akshay Sharma
**Current Behavior**

@captain-Akshay has been a consistent contributor and community member.

**Desired Situation**
Let's recognize @captain-Akshay as a contributor and community member by creating a profile on https://layer5.io/community/members.

- GitHub: @captain-Akshay
- Twitter: [@akshay_0503](https://twitter.com/akshay_0503)
- LinkedIn: [akshay-sharma-76a267203](https://www.linkedin.com/in/akshay-sharma-76a267203/)
- Meshery Cloud: 
- Link to profile picture:

A detailed explanation of how to set up a community member profile can be found in the [CONTRIBUTING.md](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

---

**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](https://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by shishiro26**
**Description**

This PR fixes #5033 

**Notes for Reviewers**
added akshay sharma to the members list 
image

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 334d49ace6450e939c39d61a85adb3908d6918e0 at: https://6528f3f57aab9b7550450c33--layer5.netlify.app

File: src/collections/members/akshay-sharma/akshay-sharma.jpeg
Comment (abhijeetgauravm) [inline src/collections/members/akshay-sharma/akshay-sharma.jpeg:1]: Hey @shishiro26 can we use .webp format here?

Comment (shishiro26) [reply src/collections/members/akshay-sharma/akshay-sharma.jpeg:1]: ok I will change image to webp

**Comment by l5io** [comment]
🚀 Preview for commit d0e0932d3f5909fc385378c19b9da0bf76a7df5f at: https://6528fb9b34fba107a07e1065--layer5.netlify.app

**Review by abhijeetgauravm — APPROVED** [review]
Thanks for the contribution @shishiro26 LGTM!

**Comment by abhijeetgauravm** [comment]
@captain-Akshay your review would be appreciated!

**Comment by captain-Akshay** [comment]
LGTM! @abhijeetgauravm  & @shishiro26 Thanks again !

**Comment by abhijeetgauravm** [comment]
> LGTM! @abhijeetgauravm & @shishiro26 Thanks again !

// @ritiksaxena124 @sudhanshutech

**Comment by ritiksaxena124** [comment]
LGTM 🚀

**Pull request merged**